README.md

```diff
@@ -25,14 +25,14 @@ If you need help using or installing Node.js, please use the
 * [Contributing to the project](./CONTRIBUTING.md)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
-### Community Resources
+### Unofficial Resources
 
 * IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
 see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
 
-*Please note that community provided resources are neither managed by -- nor
-necessarily endorsed by -- the Node.js TSC/CTC. Specifically, such resources are
-not currently covered by the [Node.js Moderation Policy][] and the selection and
+*Please note that unofficial resources are neither managed by -- nor necessarily
+endorsed by -- the Node.js TSC/CTC. Specifically, such resources are not
+currently covered by the [Node.js Moderation Policy][] and the selection and
 actions of resource operators/moderators are not subject to TSC/CTC oversight.*
```

Comment:
Similar nit: `TSC/CTC oversight` -> `oversight by the Node.js project` or similar.

(These are both take-it-or-leave-it nits. If you feel like it's important to specify TSC/CTC here, then OK, that's good enough for me, leave it as is! I just want to make sure it's an intentional "there's a good reason to specificy TSC/CTC here" and not "eh, I just default to typing TSC/CTC...".)